aikau/src/main/resources/alfresco/lists/AlfHashList.js

```diff
@@ -194,15 +196,18 @@ define(["dojo/_base/declare",
          this.inherited(arguments);
          if (this.mapHashVarsToPayload)
          {
-            var hashString = hash();
```

Comment:
Is it worth replacing other uses of hash() with a hashUtils call (and then removing the dependency), to keep things consistent?